CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
     * `server.grpc-max-concurrent-streams` (for both query-frontends and queriers)
 * [CHANGE] The frontend http server will now send 502 in case of deadline exceeded and 499 if the user requested cancellation. #2156
 * [CHANGE] Config file changed to remove top level `config_store` field in favor of a nested `configdb` field. #2125
+* [CHANGE] We now ignore queries that are more than 10mins into the future. #1929
+  * Can be configured using `querier.max-query-into-future`.
```

Comment:
I'm wondering if it would be more clear:

```suggestion
* [CHANGE] We now enforce queries to be up to `-querier.max-query-into-future` into the future (defaults to 10m). #1929
```

Because we don't "ignore queries" but we actually clamp the max timestamp.